### DISCUSSION TITLE
Is it possible to not show comments on my site but show them in Github Discussions?

### DISCUSSION THREAD

**Discussion by syeda-git** [giscus/giscus#1468] [Q&A]
Hi,
Teams would like to hear feedback from users but they don't want feedback to show on the site. They are fine if the feedback displays in Discussions. Is there a way to do it?

Thanks...

**Comment by laymonage** [comment]
You could hide the comments via CSS, but that's hacky and doesn't prevent the discussions data from actually being loaded. I think you're better off writing your own solution.
